docs/cli.adoc

```diff
@@ -0,0 +1,182 @@
+:showtitle:
+:toc: left
+:icons: font
+
+= Oxide CLI
+
+The `oxide` CLI is used to access Nexus' external API, which is
+the public interface to Omicron.
+
+For more detail, refer to https://docs.oxide.computer/cli/manual[`oxide`'s CLI Manual].
```

Comment:
In the rendered version (https://github.com/oxidecomputer/omicron/blob/c6ec4d0c919619d4c2bf133b61385ce096b2793f/docs/cli.adoc#oxide-cli), I see a stray backtick. I think the backticks around "oxide" on this line can be removed

Reply:
Removed